components/image/graphviz/graphviz.p5m

```diff
@@ -44,7 +44,9 @@ file path=usr/bin/diffimg
 file path=usr/bin/dijkstra
 file path=usr/bin/dot
 link path=usr/bin/dot2gxl target=gxl2gv
-file path=usr/bin/dot_builtins
+# Prevent missing dependency
+#   depend type=require fmri=__TBD pkg.debug.depend.file=libgvplugin_vt.so.6
```

Comment:
I assume the `libgvplugin_vt.so.6` file is created during the `graphviz` build and so it should be added to a package.

Reply:
Yes, the file is part of graphviz but gmake sample-manifest doesn't include it.

Reply:
Hm, it looks like it is already there. I need to investigate further...

Reply:
It is in the sample manifest, but not in a real manifest used to publish the component.

Reply:
```
$ find components/image/graphviz/ -name '*.p5m' | xargs grep _vt
components/image/graphviz/manifests/sample-manifest.p5m:link path=usr/lib/$(MACH64)/graphviz/libgvplugin_vt.so \
components/image/graphviz/manifests/sample-manifest.p5m:    target=libgvplugin_vt.so.6.0.0
components/image/graphviz/manifests/sample-manifest.p5m:link path=usr/lib/$(MACH64)/graphviz/libgvplugin_vt.so.6 \
components/image/graphviz/manifests/sample-manifest.p5m:    target=libgvplugin_vt.so.6.0.0
components/image/graphviz/manifests/sample-manifest.p5m:file path=usr/lib/$(MACH64)/graphviz/libgvplugin_vt.so.6.0.0
$
```

Reply:
Yes, I noticed it. There is another plugin library missing which I have added now.